spec/support/private_key_file_helper.rb

```diff
@@ -0,0 +1,25 @@
+module PrivateKeyFileHelper
+  # Returns the private key in AppArtifacts.store.oidc_private_key if
+  # Identity::Hostdata.in_datacenter? or if the private key file does
+  # not exist; otherwise, the private key from the file is returned.
+  def private_key_from_store_or(file_name:)
+    file_name = force_tmp_private_key_file_name file_name: file_name
+
+    raise "Private key file '#{file_name}' not found!" if Rails.env.test? && !File.exist?(file_name)
+
+    if Identity::Hostdata.in_datacenter? || !File.exist?(file_name)
```

Comment:
`Identity::Hostdata.in_datacenter?` is meant to be like "are you in production or a deployed environment" so it should never be true in specs, is there something in particular its guarding against here?

Reply:
That was cargo-culting off of (I believe) Hooper's script. All we need for the test is a private key, so I can remove that assuming that `AppArtifacts.store.oidc_private_key` is okay to use in any test env?